File: bm-a/src/main/java/com/oayilix/lodestar/demo/modulea/SecondActivity.java
package com.oayilix.lodestar.demo.modulea;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;

import com.oayilix.lodestar.annotations.Destination;

@Destination(url = "router://example.com/app/second", description = "second page")
public class SecondActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_second);
    }
}

File: bm-a/src/main/kotlin/com/oayilix/lodestar/demo/modulea/SecondActivity.kt
package com.oayilix.lodestar.demo.modulea

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.oayilix.lodestar.annotations.Destination

@Destination(url = "lodestar://example.com/app/second", description = "second page")
class SecondActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_second)
    }
}
